templates/tower-project.yaml

```diff
@@ -465,30 +480,62 @@ Resources:
               - "s3:GetBucketLocation"
               - "s3:ListBucket"
               - "s3:GetObject"
+              - "s3:DeleteObject"
+            Resource:
+              - !Sub "arn:aws:s3:::${TowerScratch}"
+              - !Sub "arn:aws:s3:::${TowerScratch}/*"
+          - Sid: TowerForgeRoleS3WriteAccess
+            Effect: Allow
+            Principal:
+              AWS:
+                - !GetAtt TowerForgeServiceRole.Arn
+                - !GetAtt TowerForgeBatchHeadJobRole.Arn
+                - !GetAtt TowerForgeBatchWorkJobRole.Arn
+            Action:
               - "s3:PutObject"
               - "s3:PutObjectTagging"
-              - "s3:DeleteObject"
               - "s3:*MultipartUpload*"
             Resource:
               - !Sub "arn:aws:s3:::${TowerScratch}"
-              - !Sub "arn:aws:s3:::${TowerScratch}/*"
+              - !Sub "arn:aws:s3:::${TowerScratch}/work/*"
+              - !Sub "arn:aws:s3:::${TowerScratch}/07_days/*"
```

Comment:
These need to change

Reply:
in the latest commit, I changed the bucket directories to reflect the new number of days everywhere in the template